### PR TITLE
add documentation packs

### PR DESCRIPTION
**Pull request by nr-kkenney**
### Summary
* initial attempt at migrating integrations from https://newrelic.com/integrations

Resolves: #128

### CODE REVIEW

File: packs/prometheus/config.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name: Prometheus from newrelic.com/integration
```
Comment (aswanson-nr) [inline packs/prometheus/config.yml:1]: I think we should move the `prometheus-node-exporter-mac` under the `prometheus` umbrella

File: packs/rails/config.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+name: Rails from newrelic.com/integration
```
Comment (aswanson-nr) [inline packs/rails/config.yml:1]: This should probably be moved under the `ruby` umbrella folder

File: packs/monorail/config.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+name: MonoRail from newrelic.com/integration
```
Comment (aswanson-nr) [inline packs/monorail/config.yml:1]: This should probably be moved under the `dotnet` umbrella folder

**Review by aswanson-nr — COMMENTED** [review]
Found a few packs that could be moved under an umbrella folder

File: packs/prometheus/config.yml (hunk 1 above)
Comment (aswanson-nr) [reply packs/prometheus/config.yml:1]: This could just be done as part of https://github.com/newrelic/newrelic-observability-packs/issues/129

**Comment by aswanson-nr** [comment]
I just noticed that the `documentation` field is supposed to be an array of objects and this has them as a single object. I'm not quite sure if this matters in yaml, but I figured it would be worth calling out.

```yaml
documentation:
  name: thing
  description: stuff
  url: url.com
```
VS
```yaml
documentation:
  - name: thing
    description: stuff
    url: url.com
```

File: packs/monorail/config.yml (hunk 3 above)
Comment (nr-kkenney) [reply packs/monorail/config.yml]: moved 👍

File: packs/rails/config.yml (hunk 2 above)
Comment (nr-kkenney) [reply packs/rails/config.yml]: moved 👍

**Comment by rudouglas** [comment]
Yes that does indeed matter in yaml and that's the correct way to go:

```yaml
documentation:
  - name: thing
    description: stuff
    url: url.com
```

File: packs/ruby/rainbows!/config.yml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+name: Rainbows!
```
Comment (rudouglas) [inline packs/ruby/rainbows!/config.yml:1]: will the `!` in `packs/ruby/rainbows!` cause any issues i wonder?

**Review by rudouglas — CHANGES_REQUESTED** [review]
Just a few little things i noticed

File: packs/java/asynchttpclient/config.yml [hunk 5]
```diff
@@ -0,0 +1,16 @@
+name: ASyncHTTPClient
+description: >-
+  <p>Library that allows Java applications to easily execute HTTP requests and
```
Comment (aswanson-nr) [inline packs/java/asynchttpclient/config.yml:3]: This `<p>` and `</p>` should be removed

**Review by aswanson-nr — CHANGES_REQUESTED** [review]
I noticed one more `<p>` tag, otherwise looks good to go!

File: packs/prometheus/config.yml (hunk 1 above)
Comment (nr-kkenney) [reply packs/prometheus/config.yml]: ill leave this for 129 then

File: packs/aws/amazon-cloudwatch-metric-streams/config.yml [hunk 6]
```diff
@@ -0,0 +1,16 @@
+name: Amazon CloudWatch Metric Streams
+description: >-
+  <p>Faster telemetry from Amazon CloudWatch for all AWS Services and custom
+  namespaces without API throttling.</p>
```
[the remark this answers is not in the document]
Comment (nr-kkenney) [reply packs/aws/amazon-cloudwatch-metric-streams/config.yml]: removed

File: packs/aws/amazon-cloudwatch-metric-streams/config.yml [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+icon: icon.svg
+logo: logo.svg
+level: Community
+authors:
+  - New Relic
+documentation:
+  - name: Amazon CloudWatch Metric Streams installation docs
+    description: >-
+      <p>Faster telemetry from Amazon CloudWatch for all AWS Services and custom
+      namespaces without API throttling.</p>
```
[the remark this answers is not in the document]
Comment (nr-kkenney) [reply packs/aws/amazon-cloudwatch-metric-streams/config.yml]: removed

File: packs/aws/amazon-eks-on-aws-fargate/config.yml [hunk 8]
```diff
@@ -0,0 +1,16 @@
+name: Amazon EKS on AWS Fargate
+description: >-
+  <p>Telemetry from Kube State Metrics, Kubelet, and cAdvisor for full
+  observability of Kubernetes clusters running on EKS in Fargate.</p>
```
[the remark this answers is not in the document]
Comment (nr-kkenney) [reply packs/aws/amazon-eks-on-aws-fargate/config.yml]: removed

File: packs/aws/amazon-eks-on-aws-fargate/config.yml [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+icon: icon.svg
+logo: logo.svg
+level: Community
+authors:
+  - New Relic
+documentation:
+  - name: Amazon EKS on AWS Fargate installation docs
+    description: >-
+      <p>Telemetry from Kube State Metrics, Kubelet, and cAdvisor for full
+      observability of Kubernetes clusters running on EKS in Fargate.</p>
```
[the remark this answers is not in the document]
Comment (nr-kkenney) [reply packs/aws/amazon-eks-on-aws-fargate/config.yml]: removed

File: packs/dotnet/config.yml [hunk 10]
```diff
@@ -0,0 +1,16 @@
+name: .NET
+description: >-
+  <p>A developer platform with tools and libraries for building web, mobile,
+  desktop, games, IoT, cloud, and microservices.</p>
```
[the remark this answers is not in the document]
Comment (nr-kkenney) [reply packs/dotnet/config.yml]: removed

File: packs/dotnet/config.yml [hunk 11]
```diff
@@ -0,0 +1,16 @@
...
+icon: icon.svg
+logo: logo.svg
+level: Community
+authors:
+  - New Relic
+documentation:
+  - name: .NET installation docs
+    description: >-
+      <p>A developer platform with tools and libraries for building web, mobile,
+      desktop, games, IoT, cloud, and microservices.</p>
```
[the remark this answers is not in the document]
Comment (nr-kkenney) [reply packs/dotnet/config.yml]: removed

File: packs/java/asynchttpclient/config.yml (hunk 5 above)
Comment (nr-kkenney) [reply packs/java/asynchttpclient/config.yml]: removed

File: packs/ruby/rainbows!/config.yml (hunk 4 above)
Comment (nr-kkenney) [reply packs/ruby/rainbows!/config.yml]: removed in the config, and directory name

**Review by rudouglas — APPROVED** [review]
Bellissimo

**Pull request merged**